### ISSUE TITLE
Please share this to npm @types repository

### ISSUE THREAD

**Issue by viktor1190** [harish2704/types-feathers#1]
This is so cool, there are many developers searching for this, please open a request to upload this typescript definition on the [DefinitelyTyped project](https://github.com/DefinitelyTyped/DefinitelyTyped/labels/Definition%3ARequest) your contribution is very important for all us. I'm not expert but I will be happy to help you with whatever you need to get this code in the main npm @types repositories.

look the [main request thread](https://github.com/feathersjs/feathers/issues/381). Although maybe you already knew, its just to highlight my point

**Comment by harish2704** [comment]
Hi, Thank you for your interest and time.

Even-though I am aware about the fact that, It will be useful to many others, I am not doing that because of the following reason
* Feathers team is  already working on this part. So they will do this very soon, in more accurate maner.
* I didn't wrote these definitions. I got this from some website after googling. I just made it as a npm package
* Anyone can use this repo directly in their project using the command `npm i https://github.com/harish2704/types-feathers/archive/v0.0.1.tar.gz`
